web/src/App.svelte

```diff
@@ -98,6 +98,9 @@
 
   let map: Map | null = null;
   $: if (map) {
+    map.keyboard.disableRotation();
+    map.dragRotate.disable();
+    map.touchZoomRotate.disableRotation();
```

Comment:
Yes, thank you! I've only ever accidentally done this.

Reply:
For the record, I'll say that the map rotation is less of a problem for me. For one, I don't recall ever triggering it, and secondly, if I did, there is an obvious way to recover with the visible compass control. 

It's the "skew/perspective" control that always mystifies me, which has no obvious **visible** way to recover.

That said, I don't think I've ever really wanted to rotate the map either, so happy to see them all go away.